pom.xml

```diff
@@ -42,8 +42,8 @@
     <cmo_metadb_common.group>com.github.mskcc</cmo_metadb_common.group>
     <cmo_metadb_common.version>1.1.7.RELEASE</cmo_metadb_common.version>
     <!-- metadb server model module -->
-    <cmo_metadb_server.group>com.github.mskcc.cmo-metadb</cmo_metadb_server.group>
-    <cmo_metadb_server.version>1.1.2b.RELEASE</cmo_metadb_server.version>
+    <cmo_metadb_server.group>com.github.ao508.cmo-metadb</cmo_metadb_server.group>
+    <cmo_metadb_server.version>a028d88b7f1c7e3f815fe90f7194052be492cfe3</cmo_metadb_server.version>
```

Comment:
this will need to be updated with the upstream master repository dependency/commit hash once that pr is merged